tests/cockroachdb_import_connect_cli.robot

```diff
@@ -3,28 +3,29 @@ Documentation       To Verify Provisioning of CockroachDB Provider Account and d
 Metadata            Version    0.0.1
 
 Resource            ../resources/keywords/deploy_application.resource
-Resource            ../resources/keywords/suite_and_test_teardown.resource
 
-Suite Setup         Set Library Search Order    OpenShiftLibrary
+Suite Setup         Run Keywords
```

Comment:
@harshkumarRH do we need seleniumlibrary for CLI test scripts?

Reply:
@mrrajan  Thanks for pointing this out, it was a copy paste mistake, it should be OpenShiftLibrary
I have rectified this with latest commit

Skip If will skip the whole test suite, this is being done because CLI tests are not completely compatible with ldap user permissions